package/MDAnalysis/analysis/parallel_jobs.py

```diff
@@ -0,0 +1,244 @@
+import multiprocessing as mp
+import copy
+from operator import itemgetter
+
+# For fancy progressbar
+try:
+    from blessings import Terminal
+    from progressive.bar import Bar
+    from progressive.tree import ProgressTree, Value, BarDescriptor
+except:
```

Comment:
you should avoid bare except. Use 

```python
except ImportError:
    ...
```
https://docs.python.org/2/howto/doanddont.html#except